fpdf/table.py

```diff
@@ -0,0 +1,337 @@
+from dataclasses import dataclass
+from numbers import Number
+from typing import List, Union
+
+from .enums import Align, TableBordersLayout
+from .fonts import FontStyle
+
+
+DEFAULT_HEADINGS_STYLE = FontStyle(emphasis="BOLD")
+
+
+class Table:
+    """
+    Object that `fpdf.FPDF.table()` yields, used to build a table in the document.
+    Detailed usage documentation: https://pyfpdf.github.io/fpdf2/Tables.html
+    """
+
+    def __init__(
+        self,
+        fpdf,
+        rows=(),
+        *,
+        align="CENTER",
+        borders_layout=TableBordersLayout.ALL,
+        cell_fill_color=None,
+        cell_fill_logic=lambda i, j: True,
+        col_widths=None,
```

Comment:
It might be helpful to offer percentages here as well (we need those for `write_html()` anyway).

Reply:
Percentages are already supported 😊